Design rules/readme.md

```diff
@@ -0,0 +1,124 @@
+# Design Rules VNGRealisatie
+
+## Inleiding
+
+In de onderliggende Design Rules zijn de inzichten vastgelegd die zijn opgedaan bij het ontwerpen en specificeren van API's binnen diverse projecten waar VNG Realisatie bij betrokken is.
+Deze Design Rules zijn binnen het team Standaarden besproken en er is consensus bereikt over het feit dat deze Design Rules toegepast worden op API-specificaties die door VNR Realisatie worden gemaakt of in beheer worden genomen.
+
+Dit is een levende set van Design Rules die beïnvloed kan worden door voortschrijdend inzicht en door technologische ontwikkelingen.
+
+Om die reden kan het zijn dat de specificaties die door VNG Realisatie zijn opgesteld of in beheer zijn genomen niet voldoen aan alle Design Rules.
+Op het moment dat een nieuwe versie van een dergelijke  API-specificatie gepland wordt waarmee breaking changes worden doorgevoerd bepaalt de betreffende designer of dat een kans is om correcties door te voeren waardoor de API weer voldoet aan de onderstaande Design Rules.
+
+Een nieuwe versie van een API specificatie met een breaking change houdt dus niet automatisch in dat deze is aangepast aan de Design Rules.
+
```

Comment:
Ik zou hier nog een paragraaf wijden aan de wijze waarop VNG-R omgaat met de landelijke Design Rules en Extensions, met de eigen Design Rules en Extensions, de procedures voor het aanbrengen van wijzigingen op onze eigen DR's en Extensions en hoe we handelen als we een eigen DR willen voordragen voor de landelijke lijst.

Reply:
Dat kan. Misschien is het overzichtelijker als we dat in een apart bestand doen. Dan moeten we in de readme.md misschien alleen een korte toelichting op de opgenomen bestanden opnemen. Wil jij een voorstel voor dat bestand maken ?